### PR TITLE
Add the revised version of chat room rules to the docs

### PR DESCRIPTION
**Pull request by felix920506**
As discussed here https://github.com/jellyfin/jellyfin-meta/discussions/69 and in Matrix chat, adding the new version of rules to be part of the docs which can them be linked on Matrix, Discord and IRC.

### CODE REVIEW

File: docs/general/community-standards/chat.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Technical Misinformation Rules
+
+Since the chat rooms are much more likely to see users less experienced with tech, there are additional rules about technical misinformation specific to the chat rooms.
+
+1. Definition of Technical Misinformation: Information about technology that can be objectively proven wrong.
+
+2. You will generally **NOT** face punishment for simply saying something wrong occasionally. This rule is reserved for users who repeatedly demonstrate their unwillingness / inability to do basic research and / or take corrections from the community.
+
+3. You may **NOT** spread information that may violate the ToS (Terms of services) of other services (eg. Hosting JF behind CloudFlare). When the ToS is unclear, you should assume that it is against the ToS.
```
Comment (joshuaboniface) [inline docs/general/community-standards/chat.md:57]: Please use the full name "Jellyfin", not "JF", within official docs.

File: docs/general/community-standards/chat.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+4. Discussion of topics related to technical misinformation should be limited to #offtopic only.
+
+## Contact and Punishments
+
+1. If you see something against the rules, or something that makes you feel unsafe, let our staff know. We want this server to be a welcoming space for all users of Jellyfin.
+
+2. If you wish to contact moderators privately, please send an E-mail to [team@jellyfin.org](mailto:team@jellyfin.org) containing all the relevant info on your matter.
```
Comment (joshuaboniface) [inline docs/general/community-standards/chat.md:65]: I'd prefer this link to the Contact page, as that lists both the general email (I think?) as well as individuals. I'd have no problem with people DMing me on various platforms about moderation issues.

**Review by joshuaboniface — APPROVED** [review]
Looks good with a bit of feedback.

File: docs/general/community-standards/chat.md (hunk 2 above)
Comment (felix920506) [reply docs/general/community-standards/chat.md:65]: No, there are no E-mails listed on the contact page.

Comment (felix920506) [reply docs/general/community-standards/chat.md:65]: This is how it is listed in the community standards page
截圖 2024-04-15 下午12 09 01

Comment (felix920506) [reply docs/general/community-standards/chat.md:65]: Ok I'll make that fix in a bit

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**